etc/scripts/create-new-schema.sh

```diff
@@ -21,4 +21,5 @@ pg_dump -U uta_admin -h localhost -d uta -n "$source_uta_v" | \
 # create new schema
 gzip -cdq $dumps_dir/"$source_uta_v".pgd.gz | \
  sbin/pg-dump-schema-rename "$source_uta_v" "$dest_uta_v" | \
+ sbin/pg-dump-schema-rename "uta_1_1" "$dest_uta_v" | \
```

Comment:
Alembic automatically creates names for indexes when not given one. The temporary database schema name "uta_1_1" was imbedded in them. So I added this line to update them. The issue starts when we don't use the temporary named "uta_1_1". Alembic starts to detect differences. To solve this I propose that we do not include the schema version in the temporary name. We can just keep it simply "uta".